aws/table_aws_elastic_beanstalk_application_version.go

```diff
@@ -0,0 +1,248 @@
+package aws
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk/types"
+	"github.com/aws/smithy-go"
+
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk"
+
+	elasticbeanstalkv1 "github.com/aws/aws-sdk-go/service/elasticbeanstalk"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsElasticBeanstalkApplicationVersion(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_elastic_beanstalk_application_version",
+		Description: "AWS Elastic Beanstalk Application Version",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("application_name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException"}),
+			},
+			Hydrate: getElasticBeanstalkApplicationVersion,
+			Tags:    map[string]string{"service": "elasticbeanstalk", "action": "DescribeApplicationVersions"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listElasticBeanstalkApplicationVersions,
+			Tags:    map[string]string{"service": "elasticbeanstalk", "action": "DescribeApplicationVersions"},
+		},
+		HydrateConfig: []plugin.HydrateConfig{
+			{
+				Func: listAwsElasticBeanstalkApplicationVersionTags,
+				Tags: map[string]string{"service": "elasticbeanstalk", "action": "ListTagsForResource"},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(elasticbeanstalkv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "application_name",
+				Description: "The name of the application to which the application version belongs.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "application_version_arn",
+				Description: "The Amazon Resource Name (ARN) of the application version.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "build_arn",
+				Description: "Reference to the artifact from the AWS CodeBuild build.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "date_created",
+				Description: "The creation date of the application version.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "date_updated",
+				Description: "The last modified date of the application version.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "description",
+				Description: "The description of the application version.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "source_build_information",
+				Description: "Information about the source code for the application version if the source code was retrieved from AWS CodeCommit.",
+				Type:        proto.ColumnType_JSON,
```

Comment:
@Priyanka-Chatterjee-2000 `JSON` type column should be placed after the `string` type columns.